sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/AppendBlobAsyncClient.java

```diff
@@ -49,6 +49,8 @@
  * responses to a {@link java.util.concurrent.CompletableFuture} object through {@link Mono#toFuture()}.
  */
 public final class AppendBlobAsyncClient extends BlobAsyncClient {
+    private final ClientLogger logger = new ClientLogger(AppendBlobAsyncClient.class);
```

Comment:
No longer need this, clean up line changes in this file as well since we don't need to modify it.

Reply:
Done.